app/templates/search/_briefs_results.html

```diff
@@ -1,46 +1,46 @@
 {% for brief in briefs %}
 <div class="search-result">
     <h2 class="search-result-title">
-        <a href="{{ url_for('.get_brief_by_id', framework_framework=framework.framework, brief_id=brief.data.id) }}">{{ brief.data.title }}</a>
+        <a href="{{ url_for('.get_brief_by_id', framework_framework=framework.framework, brief_id=brief.id) }}">{{ brief.title }}</a>
     </h2>
 
     <ul class="search-result-important-metadata">
         <li class="search-result-metadata-item">
-            {{ brief.data.organisation }}
+            {{ brief.organisation }}
         </li>
         <li class="search-result-metadata-item">
-            {{ brief.data.location }}
+            {{ brief.location }}
         </li>
     </ul>
 
     <ul class="search-result-metadata">
         <li class="search-result-metadata-item">
-            {{ brief.data.lotName }}
+            {{ brief.lot.name }}
         </li>
-        {% if 'specialistRole' in brief.data %}
+        {% if 'specialistRole' in brief %}
         <li class="search-result-metadata-item">
-            {{ brief.content.summary(brief.data).get_question("specialistRole").value }}
+            {{ brief.specialistRole | capitalize() }}
         </li>
         {% endif %}
     </ul>
 
     <ul class="search-result-metadata">
-        {% if brief.data.status in ['closed', 'awarded', 'cancelled', 'unsuccessful'] %}
+        {% if brief.status in ['closed', 'awarded', 'cancelled', 'unsuccessful'] %}
```

Comment:
hmm, it's a pity we aren't ending up with dicts back from the Search API that are more directly compatible with those that come from the main API. Not something we can fix right now, but something to think about next time are looking at either of the two maybe? cc @risicle

Reply:
Yup exactly what I said.